pkg/git/git.go

```diff
@@ -18,7 +18,6 @@ import (
 	"time"
 
 	"github.com/pkg/errors"
-
```

Comment:
You have been trained well :)

Reply:
yes yes!

Reply:
this is called `reflex arc`. When I saw an empty line in import I have reflex arc to remove it :)